src/Bicep.IO/Bicep.IO.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.IO.Abstractions" Version="21.0.29" />

```

Comment:
Is this to facilitate the refactor, or something we see sticking around long-term?

Reply:
`System.IO.Abstraction` will still be around and used for mocking the file system in tests. However, I plan to restrict references to the namespace within `Bicep.Core.